scripts/multisigTransactionProposals/README.md

```diff
@@ -0,0 +1,85 @@
+```
+
+Once successful, all signers will see a transaction request on the SAFE UI e.g.
+`https://app.safe.global/transactions/queue?safe=$SAFE_MULTISIG_ADDRESS`
+
+Once the transaction has been signed by all signers and executed by one, you should be able to go to the light client
+proxy and read the permissioned prover address.
+`ts-node scripts/multisigTransactionProposals/setPermissionedProverProposal.ts <network_name> <lightClientContract> <proverAddress> <multisigWalletAddress>`
+
+## Disable Permissioned Prover
+
+To disable the permissioned prover on the light client contract, ensure that the following environment variables are set
+in the `.env` file:
+
+- `SEPOLIA_RPC_URL`
+- `ORCHESTRATOR_SIGNER_KEY`
+- `SAFE_MULTISIG_ADDRESS`
+- `LIGHT_CLIENT_CONTRACT_ADDRESS`
+
+Assuming you're in the root folder, run the following command:
+
+```bash
+ts-node scripts/multisigTransactionProposals/safeSDK/disableProverProposal.ts
+```
+
+Once successful, all signers will see a transaction request on the SAFE UI
+`https://app.safe.global/transactions/queue?safe=$SAFE_MULTISIG_ADDRESS`
+
+Once the transaction has been signed by all signers and executed by one, you should be able to go to the light client
+proxy and read the permissioned prover address. It will be equal to the 0 ETH address (address(0)).
+
+## Demonstrating the setPermissionedProver workflow
+
+1. Follow the steps in the deployment script [readme](../../contracts/script/README.md) to set up OpenZeppelin Defender,
+   a Multisig Wallet and deploy the Light Client contract
+2. Set the environment variables mentioned in the section, [Set Permissioned Prover](#set-permissioned-prover)
+3. Run the `ts-node` command as mentioned in the section, [Set Permissioned Prover](#set-permissioned-prover)
+
+## Demonstrating the disablePermissionedProver workflow
+
+1. Follow the steps in the deployment script [readme](../../contracts/script/README.md) to set up OpenZeppelin Defender,
+   a Multisig Wallet and deploy the Light Client contract
+2. Set the environment variables mentioned in the section, [Disable Permissioned Prover](#disable-permissioned-prover)
+3. Run the `ts-node` command as mentioned in the section, [Disable Permissioned Prover](#disable-permissioned-prover)
+
+## Testing
+
+### Testing Safe Multisig Wallets
+
+The Safe Transaction Service requires a live network available for testing and the current service only supports mainnet
+and testnets such as Sepolia. The Safe Wallet UI only works with public networks that they support and to customize it
+for a private EVM network, read their [docs](https://help.safe.global/en/articles/40795-supported-networks) for more
+info. It's non-trivial to set up Safe for private networks and therefore time has not been allocated to do so at this
+stage.
+
+### Testing the utils
+
+Testing safeSDK/utils.ts
+
+```bash
+yarn jest scripts/multisigTransactionProposals/tests/utils.test.ts
```

Comment:
I'm getting this error
```
yarn run v1.22.22
$ /home/lulu/r/EspressoSystems/espresso-sequencer/node_modules/.bin/jest scripts/multisigTransactionProposals/tests/utils.test.ts
● Validation Error:

  Module ts-jest in the transform option was not found.
         <rootDir> is: /home/lulu/r/EspressoSystems/espresso-sequencer

  Configuration Documentation:
  https://jestjs.io/docs/configuration

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Reply:
ah it may have been installed globally on my machine and not locally https://github.com/EspressoSystems/espresso-sequencer/commit/a247a53f75d906cce06bf1320321f21068a74786